Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -1,97 +1,88 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { TreeItem, TreeView } from '@material-ui/lab';
-import { MdChevronRight, MdExpandMore } from 'react-icons/md';
-import { isArray, isEmpty } from 'lodash';
-
-interface ITreeItemType {
-    id: string,
-    name: string,
-    parentId?: string,
-    children?: ITreeItemType[],
-}
+import { isArray, isEmpty, without } from 'lodash';
+import MdExpandMoreIcon from '../icons/MdExpandMoreIcon';
+import MdChevronRightIcon from '../icons/MdChervronRightIcon';
+import ITreeItemType from '../../../common/tree-types';
 
 interface ITreeProps {
     items: ITreeItemType[];
-    onTreeLabelClick: (node: ITreeItemType) => void;
+    onCategoryLabelClick: (node: ITreeItemType) => void;
```

Comment:
This is strange. You are inside the `ITreeProps`, i.e. props of a `Tree`, but you have a specific `onCategoryLabelClick`.